### PR TITLE
Feature/clean metro logs remove old wallet UI

### PR DESCRIPTION
**Pull request by flexsurfer**
clean metro logs remove old wallet UI

### CODE REVIEW

**Review by J-Son89 — APPROVED** [review]
🚀 🚀 🚀

File: src/status_im2/contexts/profile/login/events.cljs [hunk 1]
```diff
@@ -112,31 +112,28 @@
...
-              (switcher-cards-store/fetch-switcher-cards-rpc))))
+    (rf/merge
+     cofx
+     (cond-> {:json-rpc/call                              [{:method     "wakuext_startMessenger"
+                                                            :on-success #(re-frame/dispatch
+                                                                          [:messenger-started %])
+                                                            :on-error   #(log/error
+                                                                          "failed to start messenger")}]
+              :check-eip1559-activation                   {:network-id network-id}
+              :effects.profile/enable-local-notifications nil}
```
Comment (smohamedjavid) [inline src/status_im2/contexts/profile/login/events.cljs]: We need to dispatch the `wallet/get-accounts` event on login for the new wallet to work. 

(Before this PR, we use the response of the `wallet_getAccounts` RPC call from legacy initialization)

Comment (flexsurfer) [reply src/status_im2/contexts/profile/login/events.cljs]: thanks. done

File: src/status_im2/contexts/profile/login/events.cljs [hunk 2]
```diff
@@ -112,31 +112,29 @@
...
-              (switcher-cards-store/fetch-switcher-cards-rpc))))
+    (rf/merge
+     cofx
+     (cond-> {:json-rpc/call                              [{:method     "wakuext_startMessenger"
+                                                            :on-success #(re-frame/dispatch
+                                                                          [:messenger-started %])
+                                                            :on-error   #(log/error
+                                                                          "failed to start messenger")}]
+              :check-eip1559-activation                   {:network-id network-id}
+              :wallet/get-accounts                        nil
```
Comment (smohamedjavid) [inline src/status_im2/contexts/profile/login/events.cljs]: @flexsurfer - We need to dispatch as this is an event, not an effect.

Comment (flexsurfer) [reply src/status_im2/contexts/profile/login/events.cljs]: omg this mess with reg-event-fx drives me crazy :(

Comment (flexsurfer) [reply src/status_im2/contexts/profile/login/events.cljs]: thanks done

**Review by smohamedjavid — APPROVED** [review]
Looks good! Thanks @flexsurfer 🚀

**Pull request merged**